shared/middleware/files_api.rb

```diff
@@ -723,18 +720,12 @@ def files_put_file(encrypted_channel_id, filename, body)
     # read the manifest
     bucket = FileBucket.new
     manifest_result = bucket.get(encrypted_channel_id, FileBucket::MANIFEST_FILENAME)
-    return {filesVersionId: ""}.to_json if manifest_result[:status] == 'NOT_FOUND'
+    not_found if manifest_result[:status] == 'NOT_FOUND'
     manifest = JSON.load manifest_result[:body]
 
-    abuse_score = StorageApps.get_abuse(encrypted_channel_id)
```

Comment:
It looks like we aren't otherwise setting this abuse score?

Reply:
It was only being used when rewriting the manifest. Since we're deleting the manifest, we don't need to do that.